libgloss/arc/hl/configure

```diff
@@ -0,0 +1,3975 @@
+#! /bin/sh
```

Comment:
Is it possible to get rid of this generated file and have it generated during the build process?

Reply:
As I can see all newlib's configure files are pregenerated and pushed to git.